src/components/menu/__tests__/editor-mode-toolbar.test.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable import/no-named-as-default */
+import React from 'react'
+import { shallow } from 'enzyme'
+import { EditorModeToolbarUnconnected } from '../editor-mode-toolbar'
+import EditorModeTitle from '../editor-mode-title'
+import ViewControls from '../view-controls'
+import IodideLogo from '../../../shared/iodide-logo'
+
+describe('<EditorModeToolbarUnconnected', () => {
+  let wrapper
+  beforeEach(() => {
+    wrapper = shallow(<EditorModeToolbarUnconnected />)
+  })
+  it('Should render EditiorModeTitle', () => {
```

Comment:
what are you actually testing with this "it" block? looks like it's more than EditiorModeTitle, right?

this description should be something like: "should render all child components correctly"  ... but why are you only testing 3 of the 4 components? EditorModeControls is missing.